tools/pipeline/Main.cpp

```diff
@@ -73,6 +80,16 @@ static alias SilenceAlias1("s",
                            aliasopt(Silence),
                            cat(PipelineCategory));
 
+static alias SilenceAlias2("quiet",
+                           desc("Alias for --silent"),
+                           aliasopt(Silence),
+                           cat(PipelineCategory));
+
+static alias SilenceAlias3("q",
+                           desc("Alias for --silent"),
+                           aliasopt(Silence),
+                           cat(PipelineCategory));
```

Comment:
Why?

Reply:
using revng translate --verbose yields a output that makes sense for translate but not for revng pipeline
this allows to do 
```
revng translate -- -q
revng translate -- --verbose
```
when one is debugging the pipeline itself

Reply:
In an unrelated branch, I'm dropping `--`. I don't think anyone ever used it and if you're running multiple commands it's not really obvious who is gong to receive what comes after `--`.
If you want to debug things, you've better using `--verbose`, copy paste and customize invocation.

Reply:
--verbose is unusable on translate to do debugging about the pipeline. it runs the pipeline when you actually just want to see how it is being invoked. 
that one time when antonio tried out the pipeline it would have found helpfull multiple time in a day to have --. 
i myself quite use the -- mechanism to debug it.

i am very against droping -- in translate.